src/Microsoft.DotNet.ApiSymbolExtension/AssemblyLoadWarning.cs

```diff
@@ -3,9 +3,8 @@
 
 using System;
 using System.Collections.Generic;
-using Microsoft.DotNet.ApiCompatibility.Abstractions;
 
-namespace Microsoft.DotNet.ApiCompatibility
+namespace Microsoft.DotNet.ApiSymbolExtension
```

Comment:
Since there are multiple extensions to Roslyn in this project, I suggest this be renamed to `ApiSymbolExtensions`. It would also be similar to other places, for example in ApiCompat, we have `SymbolExtensions`.

Reply:
Updated the namespace and project files.